src/daemons/publishDaemon.ts

```diff
@@ -76,7 +83,9 @@ async function sendToDevice(
       },
       data: message.data ?? {}
     })
-  } catch (err) {}
+  } catch (error) {
+    logger.info('Unknown error', { deviceId, error })
+  }
```

Comment:
As much of a pain as it is, the refactor commit where `sendToDevice` was implement in `pushSender.ts` doesn't have this fixup. Optional.

Reply:
This error handling is a new feature, just added in this commit. The code was always `catch (e) {}` before.